### ISSUE TITLE
Inappropriate default alt text for table icons

### ISSUE THREAD

**Issue by vaadin-bot** [vaadin/framework#4997]
**Originally by @Legioth**

___
As discussed in https://dev.vaadin.com/review/#/c/2883/, table uses an inappropriate default alt text for icons. It should be changed to use "" instead of "icon".

___
 Imported from https://dev.vaadin.com/ issue #13479
